src/app/(proper_react)/redesign/(authenticated)/user/dashboard/page.tsx

```diff
@@ -13,12 +13,23 @@ import { headers } from "next/headers";
 import { View } from "./View";
 import { getUserBreaches } from "../../../../../functions/server/getUserBreaches";
 import { getLocale } from "../../../../../functions/server/l10n";
+import { getOnerepProfileId } from "../../../../../../db/tables/subscribers";
+import { authOptions } from "../../../../../api/utils/auth";
+import { getLatestOnerepScan } from "../../../../../../db/tables/onerep_scans";
 
 export default async function DashboardPage() {
-  const session = await getServerSession();
+  const session = await getServerSession(authOptions);
+  if (!session?.user?.subscriber?.id) {
+    throw new Error("No session");
+  }
   const headersList = headers();
   const countryCode = getCountryCode(headersList);
 
+  const result = await getOnerepProfileId(session.user.subscriber.id);
+  const profileId = result[0]["onerep_profile_id"] as number;
```

Comment:
Hm not your fault, I'd like to clean up this function so it returns a nicer result...